docs/topic/monitoring-alerting/uptime-checks.md

```diff
@@ -65,3 +65,22 @@ steps required.
  terraform state (on GCS), the uptime checks, notification channels and alert
  policies. *Nothing destructive* can happen if this `terraform apply` goes
  wrong, so it is alright to run this without human supervision on GitHub Actions
+
+(uptime-checks:snoozes)=
+ ## How do I snoooze a check?
+
+As the checks are all in GCP in the [monitoring console](https://console.cloud.google.com/monitoring/alerting?project=two-eye-two-se) 
+
```

Comment:
Is this break line intended? Also, it seems the phrase needs a continuation, maybe? (I might be my bad english as well, feel free to tell me if that is the case).